### PR TITLE
160618122 Highlight and comment functionality

### PR DESCRIPTION
**Pull request by AdinoyiSadiq**
### What does this PR do?
This pull request ensures that a user can save a highlight from an article and comment on it

#### Description of Task to be completed?
- Create a route that a user can use to save highlighted article data
- Extend the create comment route to handle highlighted article data

#### How should this be manually tested?
- Setup the repo following the README instructions
- Create an article
- use the `/articles/:slug/comments/highlight/:status` route to make a post request with a payload of `{
  "body": "Hey there, I really enjoyed reading your article",
  "pageId": "randomString",
  "highlight": "This is the text",
  "startIndex": 1,
  "endIndex": 10
}`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#160618122](https://www.pivotaltracker.com/story/show/160618122)

#### Screenshots (if appropriate)
N/A

#### Questions:
None

### CODE REVIEW

File: controllers/commentController.js [hunk 1]
```diff
@@ -13,18 +13,27 @@ const profileAtrributes = ['username', 'bio', 'image'];
 exports.postComment = async (req, res) => {
   try {
     const values = utility.trimValues(req.body);
```
Comment (tonyguesswho) [inline controllers/commentController.js:22]: I think you can add a brief comment to this section in other to describe what this code does.

Comment (AdinoyiSadiq) [reply controllers/commentController.js:22]: I see, thanks.

**Review by tonyguesswho — APPROVED** [review]
LGTM

**Review by Phunmbi — APPROVED** [review]
LGTM

**Pull request merged**